test_data/frontend/500.sql

```diff
@@ -1,3 +1,17 @@
+/* Copyright 2020 Google LLC
```

Comment:
Please check by running these files to ensure the SQL files are running after adding the comments

Reply:
Verified that all the sql files are getting parsed with these comments.